### PR TITLE
Components

### PR DESCRIPTION
**Pull request by AlejandroNeira98**
## In this part of the project the main changes were: 

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches this design:

<p align="center">
  Calculator
</p>

- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

**Review by Meltrust — COMMENTED** [review]
Hi @AlejandroNeira98 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters failing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import InputCalculator from './components/Calculator';
 
 function App() {
```
Comment (Meltrust) [inline src/App.js]: According to the project requirements:

image

Therefore:

- [x] Please, kindly make this component `App.js` to be class-based.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import InputCalculator from './components/Calculator';
```
Comment (Meltrust) [inline src/App.js]: According to the project requirements:

image


- [x] `Calculator` should be the component that gets imported into `App.js`, not `InputCalculator`.  While it's understood you are doing this in another way, please kindly correct this so the project is constructed according to instructions.

File: src/App.js (hunk 1 above)
Comment (Meltrust) [inline src/App.js]: According to the project requirements:

image

However, I'm getting this (just a tiny button):

image

This seems to be the source of the rendering problem:

image

The prop `symbol` is being read as `undefined`.

Therefore:

- [x] Kindly make sure the calculator renders in full and matches the given design.

That way, we will move closer to approval 💪

File: src/App.js (hunk 2 above)
Comment (Meltrust) [inline src/App.js]: - [ ] Nice start on your readme.  However, all the sections from the [Microverse readme template](https://github.com/microverseinc/readme-template) are required, except for "live demo" and "tests" when not required for the project.  Kindly please add all the missing sections on your readme so your app is well documented.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AlejandroNeira98,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Alejandro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**